src/runtime-config/dto/runtime-config.dto.ts

```diff
@@ -0,0 +1,36 @@
+import { ApiProperty } from "@nestjs/swagger";
+import { IsString, IsOptional, IsObject } from "class-validator";
+
+export class OutputRuntimeConfigDto {
+  @ApiProperty({
+    type: String,
+    description: "Unique config identifier (e.g. 'frontend', 'backend', etc.)",
+    example: "frontend",
+  })
+  @IsString()
+  _id: string;
```

Comment:
I suggest `key` for the unique identifier. `_id` should only be used for the internal mongo document id (which should not be accessible in the API).

Reply:
Does it make sense to have more than "frontend" and "backend"? I would make this an enum. Or are you thinking there might be use cases where other microservices store most of their configs on the scicat backend?